pkg/test/framework/test.go

```diff
@@ -198,6 +198,20 @@ func (t *Test) runInternal(fn func(ctx TestContext), parallel bool) {
 		return
 	}
 
+	// TODO: should we also block new cases?
+	var myGoTest *testing.T
+	if t.goTest != nil {
+		myGoTest = t.goTest
+	} else {
+		myGoTest = t.parent.goTest
+	}
+	suiteName := t.s.settings.TestID
+	if len(t.featureLabels) < 1 && !features.GlobalWhitelist.Contains(suiteName, myGoTest.Name()) {
```

Comment:
should we warn if there is something in whitelist that *does* have the label? that way we can clean it up?

Probably not fail though